### DISCUSSION TITLE
best score model만 저장하는 코드

### DISCUSSION THREAD

**Discussion by jiwoo0212** [boostcampaitech2/object-detection-level2-cv-06#68] [Q&A]
best score model만 저장하는 코드 어떻게 사용하고 계신가요?

아래와 같이 fromfile로 불러온 다음 변경하는 방식은 아는데(제공받은 baseline 방식)
```
cfg = Config.fromfile('./configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py')
cfg.evaluation['save_best'] = 'bbox_mAP_50'
```
config.py 에 넣어주는 방식(dict()로 넣는 방식)을 모르겠어서요..!

evaluation 코드에 오류가 났는지 확인하려면 한 에폭을 돌아야하네요...^^...

**Comment by JiyouSeo** [comment]
config.py에 evaluation = dict(interval=1, metric='bbox', save_best='bbox_mAP_50')를 추가해보시겠어요?

**Reply by jaehwan-AI** [reply]
@JiyouSeo  save_best='bbox_mAP_50'을 추가하면 학습이 안끊기고 잘 되나요?

**Reply by JiyouSeo** [reply]
재확인 한번 해보고 말씀드리겠습니다.

**Reply by JiyouSeo** [reply]
음...validation에서 key error가 뜨네요.. 좀 더 찾아봐야할 것 같습니다.

**Comment by ppskj178** [comment]
evaluation = dict(interval=1, metric='bbox', save_best='bbox_mAP_50')
이렇게 하면 안되셨나요...?
일단 저는 돌아는가는데...

**Reply by jaehwan-AI** [reply]
제 기억에는 한 에폭 돌고 다음 에폭에서 key error가 나오는 것으로 알고 있습니다

**Reply by ppskj178** [reply]
> 제 기억에는 한 에폭 돌고 다음 에폭에서 key error가 나오는 것으로 알고 있습니다

잘되는데요...?

피어세션 시간에 말씀드릴게요

**Reply by jaehwan-AI** [reply]
오 잘 되면 다행이네요
